### PR TITLE
updates for router documentation

### PR DESCRIPTION
**Pull request by pweil-**
This update includes information on setting a default certificate for a router  (used for wildcard certificates), examples of the openshift ex router command, and some notes for administrators.

/cc @ramr - the HA setup is ready to be updated when you get a chance.

PTAL

### CODE REVIEW

**Comment by danmcp** [comment]
@adellape @CowboysFan Review please

File: architecture/core_objects/routing.adoc [hunk 1]
```diff
@@ -316,23 +345,29 @@ available in OpenShift.
...
-can find the `osc` executable on the path. If it cannot find the executable, it
-creates the JSON file and displays the location. You can then manually run the
-create command.
+----
+Examples:
+  Check the default router ("router"):
 
-.Running the Router Script
-====
+  $ openshift ex router
```
Comment (detiber) [inline architecture/core_objects/routing.adoc]: Shouldn't we be referring to openshift admin router (or osadm router) here instead of openshift ex router?

Comment (pweil-) [inline architecture/core_objects/routing.adoc]: Yes, they're functionally equivalent,  I just always used openshift ex.  Updating and updating my brain cache to save 7 chars every day.  I type this a lot!

**Comment by pweil-** [comment]
Updated to use osadm and also remove the `--create` syntax which was deprecated with a PR yesterday https://github.com/openshift/origin/pull/2019  Added reference of the new `--dry-run` flag that was also in the PR

File: admin_guide/router.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
+= Routers
```
Comment (adellape) [inline admin_guide/router.adoc]: Would it make sense to title this "Monitoring Routers" / `monitoring_routers.adoc`?

**Comment by adellape** [comment]
@pweil- I'm OK with merging this as-is and I can follow up in the related docs card (https://trello.com/c/Zxpf1DPR), unless you were waiting for feedback from @ramr per the above cc.

**Comment by ramr** [comment]
@adellape - I can have look if you need me too. But the CC was just to let me know that the HA setup needs to be documented/added (that's in separate PR: 337).

**Comment by pweil-** [comment]
nope,all good if it's ok with you.

**Pull request merged**